src/vfs.js

```diff
@@ -74,6 +74,12 @@ const handleDirectoryList = (path, options) => result =>
       filter: options.filter
     }));
 
+// Returns a new "options" object without properties from "ignore"
+const filterOptions = (ignore, options) => {
+  ignore.forEach(item => delete options[item]);
+  return options;
+};
```

Comment:
You're mutating the `options` passed into a function. This is a very dangerous thing to do, because this affects what is then passed on to `handleDirectoryList`.

This will instead create a new object with the stuff filtered:

```
const filterOptions = (ignore, options) => Object.fromEntries(
  Object
    .entries(options)
    .filter(([k] => !ignore.contains(k)))
)
```

Reply:
Sorry for delay. I was on weekend :)
`filterOptions` is changed in new commit.